models/engine/db_storage.py

```diff
@@ -74,3 +74,25 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """Retrieve an object"""
+        if cls is not None and type(cls) is str and id is not None and\
+           type(id) is str and cls in classes:
```

Comment:
Could you instead of modifying the `cls` variable directly, check if it's a string and then convert it to the corresponding class. This can help avoid unintentional variable name conflicts.

- like this: 

`if isinstance(cls, str) and isinstance(id, str) and cls in classes`